doc/manpages/qvm-features.rst

```diff
@@ -449,6 +449,18 @@ feature until it is deleted.
 | **Type**: `int`
 | **Default**: `0`
 
+preload-dispvm-threshold
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Memory size in MiB to deduct (on calculation only) from system's available
+memory. The resulting value is considered before trying to preload to avoid
+exhausting system's memory.
```

Comment:
This description isn't quite clear what that feature does. Maybe better something like "Memory size in MiB to leave free when preloading disposables. If preloading a disposable would leave less memory, it's skipped".

@andrewdavidwong any better idea?